src/test/ActionTest.java

```diff
@@ -0,0 +1,29 @@
+import duke.action.Action;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ActionTest {
+    @Test
+    public void getStatus_unmarkedAction_singleWhiteSpace() {
+        Action action = new Action("Action");
+        String result = action.getStatus();
+        assertEquals(" ", result);
+    }
+
+    @Test
+    public void testSetDone() {
```

Comment:
```suggestion
    public void getStatus_markAction_xReturned() {
```
test function name could be improved into the 3 part `featureUnderTest_testScenario_expectedBehaviour`